style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
+
+This page outlines the style guide for SCSS for the jQuery Chassis project.
+
+## Comments
+
+SCSS comments should follow the valid CSS comment guidelines outlined in the jQuery contribute CSS style guide.
```

Comment:
I guess it was my mistake mistake then, my notes on that meeting for that day mentioned converting comments to valid css comments, which must have been come from the Typography PR discussion.  I remembered the first half of the Typography PR discussion about comments and wanting to align as closely as possible to valid CSS, but not the second half.  Since you and @MichaelArestad did discuss other uses cases for different comments, perhaps it would be good to bring it up in the next meeting, so the group can iron out the details on how we want to handle comments for @kristyjy to incorporate into the Style Guide.

As for the PR on comments, I had that item written down on the agenda for meetings asking the community to review it for a few weeks weeks before it was actually merged in.  However if we decide we want to go with different commenting rules for SCSS than for CSS, then we can definitely set up an issue to make everything compliant with this completed style guide.